types/2020-08-27/Invoices.d.ts

```diff
@@ -9,7 +9,7 @@ declare module 'stripe' {
       /**
        * Unique identifier for the object.
        */
-      id: string;
+      id?: string;
```

Comment:
Hi, just wondering how could an Invoice not have an `id`? Didn't see this mentioned in the migration notes https://github.com/stripe/stripe-node/wiki/Migration-guide-for-v10

Reply:
Hey @rwaller-cazoo! The Invoice object is shared with upcoming invoice endpoint, where the ID is optional. I will add a note to the migration guide.